### ISSUE TITLE
"Articluation" between Metacarpal 1 and Metacarpal 2

### ISSUE THREAD

**Issue by tlvandeest** [spawaskar-cora/cora-docs#511]
Similar to the ribs (#493), it would be good to be able to see an association/"articulation" between MC1 and MC2 when looking at each of these elements, rather than having to track the association via multiple articulations of tarsals.  Right now, there is no way to see that a MC1 and MC2 are associated with one another when on the page for each element. 
@fedamann @clegarde @SachinPawaskarUNO

**Comment by clegarde** [comment]
I still would like to push for "Morphology" as an Association that could be used for cases like these.

**Comment by emilystreetman** [comment]
I think @TarawaProject would use Morphology too

**Comment by skorps** [comment]
@SachinPawaskarUNO Can we revisit the "Morphology" option for associations?

**Comment by SachinPawaskarUNO** [comment]
@skorps Yes, Please plan on discussing this in the next CoRA SCURM meeting, which I'm hopeful will be this coming Thursday Oct 17th.
@fedamann Let's plan on having our next CoRA SCURM meeting this coming Thursday Oct 17th.

**Comment by SachinPawaskarUNO** [comment]
@clegarde @skorps @fedamann Can we please assign someone to create a list/spreadsheet of allowable/possible morphology associations (similar to articulations). This is needed to work on the specimen morphology associations.

**Comment by clegarde** [comment]
@SachinPawaskarUNO, Attached is a list of bone associations based on morphology. No restrictions on bone side are necessary.

General overview of associations in spreadsheet:
-> ribs with ribs
-> verts with verts
-> metacarpals with metacarpals
-> metatarsals with metatarsals
-> teeth with teeth
-> bone to same bone (ex: proximal tibia with distal tibia - cannot refit, but morphology is consistent and indicates they go together - just happened in recent case)

[CoRA_Associations_Morphology.xlsx](https://github.com/spawaskar-cora/cora-docs/files/3887439/CoRA_Associations_Morphology.xlsx)
